hotness/consumers.py

```diff
@@ -276,88 +276,74 @@ def _handle_anitya_update(self, upstream, package, msg):
                               "Skipping scratch build.")
                 return
 
-            cwd = os.getcwd()
-            result_rh = 0
-            rh_stuff = {}
-            tmp = tempfile.mkdtemp(prefix='thn-rh', dir='/var/tmp')
```

Comment:
This function was (and still is) _way_ too large and far too complex. I can see at least 3 nested ``try`` statements. I'd really like to see this broken into several functions.